src/modules/map/components/olMap/handler/measure/OlMeasurementHandler.js

```diff
@@ -26,6 +25,7 @@ import { isEmptyLayer } from '../../olMapUtils';
 import { emitNotification, LevelTypes } from '../../../../../../store/notifications/notifications.action';
 import { OlSketchHandler } from '../OlSketchHandler';
 import { MEASUREMENT_LAYER_ID, MEASUREMENT_TOOL_ID } from '../../../../../../plugins/MeasurementPlugin';
+import { simulateMouseEvent } from '../../../../../../../test/modules/map/components/olMap/mapTestUtils';
```

Comment:
Oops, this is a test-only util function. If we really need it, we have to think about reimplementing (and testing) it in the `src` directory